### PR TITLE
updated SSO images

### PR DESCRIPTION
**Pull request by jerae-duffin**
Update SSO images and content

### CODE REVIEW

File: single-sign-on/index.md [hunk 1]
```diff
@@ -23,46 +18,78 @@ To enable SSO in Docker Hub, you need the following:
...
+
+    ![SSO Azure1](images/sso-azure1.png){:width="500px"}
+
+4. In the Identity Provider Set Up, copy the **Redirect URL / Reply URL**.
+
+    ![SSO Azure2](images/sso-azure2.png){:width="500px"}
+
+5. Log into your IdP to complete the IdP server configuration process. Refer to your IdP documentation for detailed instructions.
+
+> **Note:**
```
Comment (usha-mandya) [inline single-sign-on/index.md:74]: The Note here resets the numbered list. Indenting the note should fix this issue

File: single-sign-on/index.md [hunk 2]
```diff
@@ -76,10 +103,15 @@ To verify ownership of a domain, add a TXT record to your Domain Name System (DN
 
 3. After you have updated the fields, click **Save**.
 
-    Note: It can take up to 72 hours for DNS changes to take effect,  depending on your DNS host. The Domains table will have an Unverified status during this time.
+> **Note:**
```
Comment (usha-mandya) [inline single-sign-on/index.md:106]: Indent note

File: single-sign-on/index.md [hunk 3]
```diff
@@ -89,22 +121,32 @@ After you’ve completed the SSO configuration process in Docker Hub, you can te
...
 
-    ![Enforced](images/sso-enforce.png){:width="500px"}
+> **Note:**
+>
+> If you want to turn off SSO and revert back to Docker’s built-in
+> authentication, click **Turn OFF Enforcement**. Your members aren’t
+> forced to authenticate through your IdP and can log into Docker using
+> their personal credentials.
+
+    ![SSO Enforced](images/sso-enforce.png){:width="500px"}
```
Comment (usha-mandya) [inline single-sign-on/index.md:138]: The image appears broken

**Review by izzychu — CHANGES_REQUESTED** [review]
Added two quick comments- updated the service account point in pre-req. And we should update 4.4.0 to 4.4.2+ (that's the one releasing today)

**Review by izzychu — APPROVED** [review]
thank you!!

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**